packages/dev-utils/src/lib/geo-location.ts

```diff
@@ -10,3 +10,109 @@ export const mockLocation: Geolocation = {
   latitude: 0,
   timezone: 'UTC',
 }
+
+const API_URL = 'https://netlifind.netlify.app'
+const STATE_GEO_PROPERTY = 'geolocation'
+// 24 hours
+const CACHE_TTL = 8.64e7
+
+// 10 seconds
+const REQUEST_TIMEOUT = 1e4
+
+interface State {
```

Comment:
We have this:

https://github.com/netlify/primitives/blob/b4cbcea15911d2adb71b3b1b36077e99bd6e7aaa/packages/dev-utils/src/lib/local-state.ts#L27